src/Pact/Types/Runtime.hs

```diff
@@ -152,6 +152,8 @@ data ExecutionFlag
   | FlagDisablePact40
   -- | Enforce key formats. "Positive" polarity to not break legacy repl tests.
   | FlagEnforceKeyFormats
+  -- | Use v0 RowData persistence
+  | FlagRowDataV0
```

Comment:
Should we flag for the new behavior or flag to return to the old behavior?

Reply:
Generally flags are new behavior.